### PR TITLE
fix(sensor): remove redundant warnings

### PR DESCRIPTION
**Pull request by alegrey91**
## Overview
This PR removes redundant logs during **cni** detection.

## Checklist before requesting a review

put an [x] in the box to get it checked 

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: e2e/controlplaneinfo_test.go [hunk 1]
```diff
@@ -41,9 +42,9 @@ var _ = Describe("ControlPlaneInfo", func() {
 			Expect(resultBody.PKIDIr.Path).To(Equal(pkiFiles.PKIDIr.Path))
 
 			// (leave it there for debugging)
-			//for i := range resultBody.PKIFiles {
-			//	fmt.Println(resultBody.PKIFiles[i].Path)
-			//}
+			for i := range resultBody.PKIFiles {
+				fmt.Println(resultBody.PKIFiles[i].Path)
```
Comment (dwertent) [inline e2e/controlplaneinfo_test.go:46]: Use the Kubescape logger instead.

File: e2e/controlplaneinfo_test.go [hunk 2]
```diff
@@ -41,9 +42,9 @@ var _ = Describe("ControlPlaneInfo", func() {
 			Expect(resultBody.PKIDIr.Path).To(Equal(pkiFiles.PKIDIr.Path))
 
 			// (leave it there for debugging)
-			//for i := range resultBody.PKIFiles {
-			//	fmt.Println(resultBody.PKIFiles[i].Path)
-			//}
+			for i := range resultBody.PKIFiles {
```
Comment (dwertent) [inline e2e/controlplaneinfo_test.go:45]: Comment out

File: sensor/CNI.go [hunk 3]
```diff
@@ -86,19 +86,16 @@ func getCNINames(ctx context.Context) []string {
...
+	}
 
-		if err != nil {
-			logger.L().Ctx(ctx).Warning("getCNIName- Failed to locate process for cni",
-				helpers.String("cni name", cni.name),
-				helpers.Error(err))
-		}
-
+	if len(CNIs) == 0 {
+		logger.L().Warning("No CNI found")
```
Comment (dwertent) [inline sensor/CNI.go:98]: What is the use case?

**Review by dwertent — CHANGES_REQUESTED** [review]

File: sensor/CNI.go (hunk 3 above)
Comment (alegrey91) [reply sensor/CNI.go:98]: Instead of returning all the waring for non-found CNIs, it returns only a single warning when no CNI have been found.

Comment (dwertent) [reply sensor/CNI.go:98]: What is the use-case for not finding CNIs? Does it happen a lot?

Comment (alegrey91) [reply sensor/CNI.go:98]: really difficult, but could happen when another **cni** (not in our list) is used or even when a **cni** is not installed, but is not a common case for production clusters.

**Comment by dwertent** [comment]
@alegrey91 I see many tests failed

**Review by dwertent — APPROVED** [review]

**Pull request merged**